interactions.js

```diff
@@ -0,0 +1,57 @@
+// test alert
+
+// alert("working");
+
+// active state of anchor links
+
+let topNavItems = document.getElementById("top-nav-items");
```

Comment:
prefer if you use `const` here and the in the next line since they're not values that we expect to change(?)